File: Software Engineering Lab/Lab 5/ufp.py
low = [3, 4, 3, 7, 5]
average = [4, 5, 4, 10, 7]
high = [6, 7, 6, 15, 10]

ei = int(input("Enter the value of EI: "))
print("\nPress 1 for low\nPress 2 for average\nPress 3 for high")
type1 = int(input("Enter the type of EI: "))

if type1 == 1:
    res1 = ei * low[0]
elif type1 == 2:
    res1 = ei * average[0]
else:
    res1 = ei * high[0]

eo = int(input("\nEnter the value of EO: "))
print("\nPress 1 for low\nPress 2 for average\nPress 3 for high")
type2 = int(input("Enter the type of EO: "))

if type2 == 1:
    res2 = eo * low[1]
elif type2 == 2:
    res2 = eo * average[1]
else:
    res2 = eo * high[1]

print("UFP = ", res1 + res2)

File: Software Engineering Lab/Lab 5/ufp.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {

    int low[] = {3, 4, 3, 7, 5};
    int average[] = {4, 5, 4, 10, 7};
    int high[] = {6, 7, 6, 15, 10};
    int EI, EO, res1, res2, UFP;
    char type1, type2;

    cout << "Enter the value of EI: ";
    cin >> EI;
    cout << "Enter the type of EI: ";
    cin >> type1;
    getchar();

    if (type1 == 'low')
        res1 = EI * low[0];
    else if (type1 == 'average')
        res1 = EI * average[0];
    else
        res1 = EI * high[0];

    cout << "Enter the value of EO: ";
    cin >> EO;
    cout << "Enter the type of EO: ";
    cin >> type2;

    if (type1 == 'low')
        res2 = EO * low[1];
    else if (type2 == 'average')
        res1 = EO * average[1];
    else
        res1 = EO * high[1];

    UFP = res1 + res2;
    cout << "UFP = " << UFP << endl;

    return 0;
}